### PR TITLE
[PBW-5959] add hasOwnProperty check to prevent prototype.js bug

### PR DESCRIPTION
**Pull request by mistermendez**

### CODE REVIEW

File: js/views/adScreen.js [hunk 1]
```diff
@@ -135,8 +135,10 @@ var AdScreen = React.createClass({
     };
 
     var playbackControlItems = [];
-    for(var item in playbackControlItemTemplates) {
-      playbackControlItems.push(playbackControlItemTemplates[item]);
+    for (var item in playbackControlItemTemplates) {
```
Comment (mityaha) [inline js/views/adScreen.js]: Wouldn't for (var item = 0; item < playbackControlItemTemplates.length; item++) be more effective?

Comment (mityaha) [inline js/views/adScreen.js]: Or is playbackControlItemTemplates an object, not array?

Comment (mistermendez) [inline js/views/adScreen.js]: right it's an object

**Comment by mityaha** [comment]
Looks good

**Pull request merged**